pkg/api/v1/types.go

```diff
@@ -2,6 +2,8 @@ package v1
 
 // TODO publish v1 to a dated api as in Azure conventions
 
+type PropertyBagKey string
```

Comment:
We will need defined constants so that the RP can use them.  I don't think it belongs in the external API though.  Perhaps in the plugin.

Reply:
Agree. But didnt knew where we want to see it so just added here as a placeholder. We can even use a string as a key, buts its suppose to be bad practice :D